### PR TITLE
Wip doc pr 46109 backport to octopus

### PR DESCRIPTION
**Pull request by zdover23**
## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/master/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "pacific"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [x] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [x] Updates relevant documentation
  - [ ] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
</details>

### CODE REVIEW

**Comment by ljflores** [comment]
The docs build failure is unrelated; waiting on the fix (#45972) to go through QA.

File: doc/rados/operations/placement-groups.rst [hunk 1]
```diff
@@ -676,4 +676,4 @@ entirely. To mark the "unfound" objects as "lost", execute the following::
 
 .. _Create a Pool: ../pools#createpool
 .. _Mapping PGs to OSDs: ../../../architecture#mapping-pgs-to-osds
-.. _pgcalc: http://ceph.com/pgcalc/
+.. _pgcalc: https://old.ceph.com/pgcalc/
```
Comment (ljflores) [inline doc/rados/operations/placement-groups.rst:679]: Cool, I didn't even know this was a thing!

**Review by ljflores — APPROVED** [review]

**Review by jdurgin — APPROVED** [review]

File: doc/rados/operations/placement-groups.rst (hunk 1 above)
Comment (zdover23) [reply doc/rados/operations/placement-groups.rst:679]: Yeah. Over the next year, I've got to improve the exposure of stuff like this.

**Comment by ljflores** [comment]
jenkins test docs

**Comment by ljflores** [comment]
jenkins render docs

**Comment by zdover23** [comment]
jenkins test docs

**Comment by zdover23** [comment]
jenkins retest this please

**Comment by ljflores** [comment]
@zdover23 the only way to re-trigger "render docs" is by rebasing the branch. Not sure why, but `jenkins render docs` doesn't seem to be working.

Try:
`git checkout wip-doc-pr-46109-backport-to-octopus`
`git pull <the name of your ceph remote repository> octopus --rebase`
`git push -f <the name of your forked ceph repository> wip-doc-pr-46109-backport-to-octopus`

**Comment by zdover23** [comment]
This is in the docs now.

**Pull request closed without merging**